### DISCUSSION TITLE
I like this idea, but is it OK to send a PR if the issue it resolves is already closed?

### DISCUSSION THREAD

**Discussion by DNin01** [ScratchAddons/ScratchAddons#4619] [Help] [answered]
(Yes, it's a new contributor! Hello!)

Specifically, I want to resolve the issue #3401 by sharing my addon, Hide Share button. That issue is closed as completed because there was [another solution](https://github.com/ScratchAddons/ScratchAddons/issues/3401#issuecomment-908722478), so I wasn't absolutely sure if a pull request resolving it would end up being merged or not. It's also a little less recent. Could I still go ahead and create a pull request to resolve the issue? And what should I do if for any reason I'm not supposed to make a pull request? I would like to see that addon, so I asked to know for sure what I can do.

**Comment by WorldLanguages** [comment, marked as answer]
Happy to hear you made an (your first?) addon!
However, the issue itself was rejected, because the motivation for hiding the Share button was simply unintentional sharing projects, and the `Confirmations` addon already handles that.
Some users might accidentally enable addons that hide critical UI elements, so we think it's always better to provide an addon that asks the user to confirm an action, instead of hiding the button that triggers the action.

**Reply by DNin01** [reply]
Makes sense, thanks for telling me. Just so you know, I was planning to add this to the editor menu bar only and keep it on the project page. Does that change the issue? And by the way, yes, this was my first addon! :)

**Reply by WorldLanguages** [reply]
That makes users less likely to think the button vanished, but I don't think it would be an addon we'd be willing to add. This level of customization (hiding specific, important buttons) can be better customized by users themselves by writing their own userstyles or even using adblocker features that let you hide elements.
